src/ucp/core/ucp_rkey.c

```diff
@@ -621,10 +621,12 @@ void ucp_rkey_resolve_inner(ucp_rkey_h rkey, ucp_ep_h ep)
     if (rkey->cache.rma_lane == UCP_NULL_LANE) {
         rkey->cache.rma_proto_index = UCP_RKEY_SW_PROTO;
         rkey->cache.rma_rkey        = UCT_INVALID_RKEY;
+        rkey->cache.max_put_short   = 0;
```

Comment:
why ? :(

Reply:
There was a problem why CI failed for all `test_ucp_rma` tests. 
Please see:
https://github.com/openucx/ucx/pull/7206/commits/e4d38c267b260185945a2a2b2d44f451b08525eb#diff-a066ef0eeea2685b85842ea6dab448561425a5b54fb86ffb3283d91afbb80dc0R280-R284

I tried relying on `rma_config->max_put_short`, but it was wrong. 
`max_put_short` should be 0 when `(worker->context->config.ext.proto_enable == 0) && (rkey->cache.rma_lane == UCP_NULL_LANE)`. Otherwise we need to add extra branch to the fastpath.